zfdg.csl

```diff
@@ -0,0 +1,384 @@
+<?xml version="1.0" encoding="utf-8"?>
+<?xml-model href="schema-master/csl.rnc" type="application/relax-ng-compact-syntax"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="note" default-locale="de-DE">
+  <!--
+================================================================================
+INFORMATIONEN ZUM ZITIERSTIL
+================================================================================
+-->
+ <info>
+    <title>Zeitschrift für digitale Geisteswissenschaften</title>
+    <title-short>ZfdG</title-short>
+    <!-- hier zoterodaten ergänzen -->
+    <id>http://www.zotero.org/styles/zfdg</id>
```

Comment:
Change to .../zeitschrift-fur-digitale-geisteswissenschaften

Reply:
Done. Change to zeitschrift-fuer-digitale-geisteswissenschaften